airflow/www/views.py

```diff
@@ -3880,63 +3880,16 @@ def action_set_retry(self, tis):
         return redirect(self.get_redirect())
 
 
-class DagModelView(AirflowModelView):
```

Comment:
Oh yeah, I think you used to be able to "edit" the DAG records but that caused all sorts of bugs, so we removed that 👍🏻